api-reference/50 Common/Object Structures/ExportPivotGridProps/customizeCell.md

```diff
@@ -4,16 +4,235 @@ type: function(options)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+<!-- %shortDescription% -->
 
 ##### param(options): Object
-<!-- Description goes here -->
+<!-- %param(options)% -->
 
 ##### field(options.excelCell): Object
-<!-- Description goes here -->
+<!-- %field(options.excelCell)% -->
 
 ##### field(options.pivotCell): ExcelPivotGridCell
-<!-- Description goes here -->
+A **PivotGrid** cell.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following code illustrates how to customize <a href="https://github.com/exceljs/exceljs#fonts" target="_blank">font</a> and <a href="https://github.com/exceljs/exceljs#alignment" target="_blank">alignment</a> in cells whose [rowType](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Pivot_Grid_Cell/#rowType) is *"D"*:
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function() {
+        $("#pivotGridContainer").dxPivotGrid({
+            // ...
+            export: {
+                enabled: true
+            },
+            onExporting(e) {
+                var workbook = new ExcelJS.Workbook();
+                var worksheet = workbook.addWorksheet('Companies');
+
+                DevExpress.excelExporter.exportPivotGrid({
+                    component: e.component,
+                    worksheet: worksheet,
+                    topLeftCell: { row: 2, column: 2 },
+                    customizeCell: function(options) {
+                        var { gridCell, excelCell } = options;
```

Comment:
It's strange that we use destructuring, but don't use constants although constants have wider support.
```suggestion
                        const { gridCell, excelCell } = options;
```